test/controllers/authors_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class AuthorsControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
```

Comment:
no point commiting these auto-generated test-files, if they are empty.
remove them all, including fixtures
we won't be using `fixtures` for testing, but `factory_bot`, more on this later